src/Components/Publishing/Sections/Sections.tsx

```diff
@@ -213,6 +293,7 @@ export class Sections extends Component<Props, State> {
             section={section}
           >
             {child}
+            {areHostedAdsEnabled && ad}
```

Comment:
I would maybe move the check for `areHostedAdsEnabled` further up to not do unnecessary work